app/views/posts/index.html.erb

```diff
@@ -1 +1,18 @@
-<h1>post index</h1>
\ No newline at end of file
+<body>
+<%= render @user %>
+<% @posts.each do |post| %>
+    <a href ="<%= user_post_path(@user, post)%>"><%= render post%></a>
+    <%= "no comments for this post" if post.comments.empty?%>
+    <div class="comments">
+    <% post.recent_comments.each do |comment|%>
+        <%=render comment%>
+    <% end %>
+    <p id="view-post"><%= link_to 'View Post', user_post_path(@user, post) %></p>
+    </div>
+    
+<div class="button-container">
+    <button>See all posts</a></button>
+</div>
+    <% end %>
```

Comment:
- [ ] @geekelo Great work in adding the views to your project, however am unable to access the user's post index page, am getting the error below, this is because you are using render and you don't have a partial view to display comments.
![image](https://github.com/geekelo/rails-blog_app/assets/61408860/6784fce4-bedb-4635-8b12-5fd33873365f)
```suggestion
<%= render @user %>
<% @posts.each do |post| %>
    <a href ="<%= user_post_path(@user, post)%>"><%= render post%></a>
    <%= "no comments for this post" if post.comments.empty?%>
    <div class="comments">
    <% post.recent_comments.each do |comment|%>
        <li> <%= comment.text %> </li>
    <% end %>
    <p id="view-post"><%= link_to 'View Post', user_post_path(@user, post) %></p>
    </div>
    
<div class="button-container">
    <button>See all posts</a></button>
</div>
    <% end %>
```